### DISCUSSION TITLE
How to Payout using this plugin

### DISCUSSION THREAD

**Discussion by Aarush-Acharya** [flutter-stripe/flutter_stripe#1740] [Q&A]
The plugin offers concise and readable documentation for accepting different forms of payment via this plugin, but I could not find documentation on how to make payout possible using this plugin

**Comment by jamesblasco** [comment]
Hi! Could you link us to the Stripe web feature you are looking for? 
Payouts can mean a lot of things.

I do not think there is a feature to pay out to people from stripe directly from a mobile or web SDK, only it is possible to receive or plan payments. 

Are you referring to this?
https://docs.stripe.com/no-code/payout

**Comment by Aarush-Acharya** [comment]
@jamesblasco  I am building an application to connect people who want to rent a car and hosts who host a car for other people to rent, I give wallet points to those hosts cutting my fees from it if some one rents a car they hosted, the hosts can then redeem these wallet points and money needs to be transferred to their bank accounts using an API I thought if stripe could receive bank payments for me, it should also be able to send money from my bank account to the hosts, if not stripe could you suggest any other third party that could do this for me

**Comment by alainjr10** [comment]
I think Stripe connect is what you are looking for. Certainly, stripe can do this, however, i don't think you can directly handle this on the front end. Not with this package at least. you'll have to write some server side logic, using the stripe server side sdks for what you want to implement.

**Comment by Aarush-Acharya** [comment]
understood thanks a lot will look into the server side sdks provided and into Stripe Connect
